uaclient/entitlements/ros.py

```diff
@@ -0,0 +1,11 @@
+from uaclient.entitlements import repo
+
+
+class ROSESMEntitlement(repo.RepoEntitlement):
+    help_doc_url = ""
+    name = "esm-ros"
+    title = "ROS ESM"
+    description = "ROS ESM Service"
```

Comment:
@lechsandecki @mitechie do we want this discription to be a bit more "descriptive"?
We have about 46 characters to play with to fit into a 80 char wide console without wrapping on a `ua status` call. Wrapping is an option too as we already let fips-updates overflow already
```bash
root@dev-i3:~# ua status
SERVICE       ENTITLED  STATUS    DESCRIPTION
cis           yes       n/a       Center for Internet Security Audit Tools
esm-infra     yes       n/a       UA Infra: Extended Security Maintenance (ESM)
fips          yes       n/a       NIST-certified core packages
fips-updates  yes       n/a       NIST-certified core packages with priority sec
urity updates
livepatch     yes       n/a       Canonical Livepatch service

Enable services with: ua enable <service>

     Account: chad.smith@canonical.com
Subscription: chad.smith@canonical.com

Reply:
I don't think it needs the UA bits to it. UA doesn't exist in the documentation at the above URL. I think "ROS Extended Security Maintenance (ESM)" works and fits with the existing text around esm-infra.

Reply:
Agree, ROS Extended Security Maintenance (ESM) should be enough. 
Once we make the announcement of UA Apps we might want to reconsider it though. (ROS ESM is part of UA A) 

In the service column, following the latest discussion on the UA channel, maybe we should have only "ros" and "ros-updates"?

Reply:
Thanks @GAN-Canonical, we'll pull this suggestion in once we have a contract update that distinguishes ros from ros-updates.